scripts/zones/Waughroon_Shrine/npcs/Armoury_Crate.lua

```diff
@@ -116,6 +116,65 @@ local loot =
         },
     },
 
+     -- BCNM Royal Jelly
```

Comment:
Assuming the rest of this script doesn't have a "pretty" formatting pattern, but would be nice to clean this up in the future.